deploy/aws/java11Exec/pom.xml

```diff
@@ -0,0 +1,62 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>poseidon</groupId>
+    <artifactId>java11Exec</artifactId>
+    <version>1.0</version>
+    <packaging>jar</packaging>
+    <name>A Java executor created for openHPI/Poseidon.</name>
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
```

Comment:
If I got the [AWS documentation](https://docs.aws.amazon.com/lambda/latest/dg/lambda-java.html) right, Java 11 is the newest version supported by AWS lambda? That is fine and will work for the all Java courses except the newest java-algorithmen2021 course. In the latest course, we switched to the latest JDK available (17.0.1 at that time) and also used language features not available with Java < 16.

Reply:
You're welcome. I also made a minor suggestion to add this finding to the documentation.